# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../isdn4k-utils/x11r7.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- isdn4k-utils-CVS-2005-10-28/xmonisdn/Imakefile.orig	2002-01-30 01:51:52.000000000 +0100
+++ isdn4k-utils-CVS-2005-10-28/xmonisdn/Imakefile	2005-10-31 14:15:25.000000000 +0100
@@ -7,6 +7,9 @@
            OBJS = xmonisdn.o Net.o
         HEADERS = NetP.h Net.h
 
+         BINDIR = /usr/X11R7/bin
+         MANDIR = /usr/X11R7/man
+
    INSTPGMFLAGS = -m 0755 -s
 
 ComplexProgramTarget(xmonisdn)
--- isdn4k-utils-CVS-2005-10-28/xisdnload/Imakefile.orig	1998-03-08 19:54:01.000000000 +0100
+++ isdn4k-utils-CVS-2005-10-28/xisdnload/Imakefile	2005-10-31 14:16:46.000000000 +0100
@@ -23,6 +33,9 @@
  OSMINORVERSION = OSMinorVersion
       DEFINES = -DOSMAJORVERSION=$(OSMAJORVERSION) -DOSMINORVERSION=$(OSMINORVERSION) -DREGEX_NUMBER
 
+         BINDIR = /usr/X11R7/bin
+         MANDIR = /usr/X11R7/man
+
 AllTarget(xisdnload)
 NormalProgramTarget(xisdnload,$(OBJS),$(DEPLIBS),$(LOCAL_LIBRARIES),NullParameter)
 
